protocol/04_detection.md

```diff
@@ -0,0 +1,55 @@
+4) The detections.csv and figures are saved into `bat-detector-msds/output_dir/recover-DATE/SD_UNIT` .
+   - The naming conventions of each file will remember recover-DATE and SD_UNIT to trace back to where the data came from.
+5) Once you hit `ENTER` on the above command, you will see this message: `nohup: ignoring input and appending output to 'nohup.out'`
+6) Use `CTRL+C` to exit the dialog and use the command `tail -f nohup.out` to view the nohup.out file.
+   - The nohup.out file will update dynamically and independently so the command will run even if you `CTRL+C` again to exit viewing.
+7) If you wish to cancel the command:
+   - Use `ps e` to view the currently running processes; the above command will be linked to python. The PID is the ID of that process displayed to the left of the process.
+   - Use `kill PID` where you type the processes' ID in place for PID to cancel the command.
+  
+## Pipeline Outputs
+1) Once the pipeline command has been run, some details will be displayed for user sanity:
+   - `Input Directory: /mnt/ubna_data_02/recover-20230609/UBNA_012`
+   - `Output CSV name: batdetect2_pipeline__recover-20230609_UBNA_012.csv`
+   - `Output Directory: output_dir/recover-20230609/UBNA_012`
+2) After these details are shown, the pipeline will search the input directory to gather a list of usable audio files to feed into batdetect2.
+   - Here, usable is defined as 30-min long audio recordings collected using the Audiomoth's internal microphone.
+   - External microphone recordings of 30-min lengths may exist in our input directories but these recordings lack real world data.
+   - Other recordings that are 488-bytes may also exist in our input directories and lack real world data.
+   - We are still investigating why these "unusable" files are generated by the Audiomoth but our pipeline ignores them using os.stat() and exiftool.
+3) The pipeline will then display the number of usable files out of the total number of files in the input directory.
+4) The pipeline will then begin generating segments for all usable files. This may take some time but still is a fraction of the time it will take to generate detections.
+5) Once segments have been generated, the pipeline will begin displaying a loading bar:
+   - Here is an example: `0%|          | 1/7560 [0:00:05<9:29:27,  4.55s/it]`
+   - Here, 0% is the rounded percent of segments whose detections have been generated.
+   - 1 is the number of segments whose detections have been generated.
+   - 7560 is the total number of segments that were generated by the pipeline.
+   - 0:00:05 is the time that has passed for generating detections. (HH:MM:SS)
+   - 9:29:27 is the time the pipeline still needs to generate detections for all segments. (HH:MM:SS)
+   - 4.55s/it is the time it takes to generate detections per segment for our pipeline on our chosen machine.
```

Comment:
For the first value, could you include a range of time that you would expect this to be in.

Could you also add a note about what a good value for the bottom value looks like and what a "bad" value for it may look like. And could you add a note about what to do if this value is "bad"?